fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/protocol/ProtocolMetricImpl.java

```diff
@@ -20,13 +22,41 @@
 
     protected final ProtocolSupport protocolSupport;
 
+    protected List<LogicalMetric> dependentMetrics;
+
+    /**
+     * Constructor.
+     *
+     * @param logicalMetricInfo  The metadata for the metric
+     * @param templateDruidQuery  Query the metric needs
+     * @param calculation  Mapper for the metric
+     * @param dependentMetrics Metrics from which this metric depend
+     */
+    public ProtocolMetricImpl(
+            @NotNull LogicalMetricInfo logicalMetricInfo,
+            @NotNull TemplateDruidQuery templateDruidQuery,
+            ResultSetMapper calculation,
+            List<LogicalMetric> dependentMetrics
+    ) {
+        this(
+                logicalMetricInfo,
+                templateDruidQuery,
+                calculation,
+                DefaultSystemMetricProtocols.getStandardProtocolSupport(),
+                dependentMetrics
+        );
+    }
+
     /**
      * Constructor.
      *
      * @param logicalMetricInfo  The metadata for the metric
      * @param templateDruidQuery  Query the metric needs
      * @param calculation  Mapper for the metric
+     *
+     * @deprecated User {{@link #ProtocolMetricImpl(LogicalMetricInfo, TemplateDruidQuery, ResultSetMapper, List)}}
```

Comment:
Git is displaying this weirdly with an underline. does that mean its not referencing the right arg types?